README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-	- A [**phantom stock**](http://www.investopedia.com/articles/stocks/12/introduction-phantom-stock.asp) award entitles you to a payment equal to the value of a share of the company's stock, upon the occurrence of certain events.
+- **Phantom equity** is a type of compensation award that references equity, but does not entitle the recipient to actual equity in the business. These awards come under a variety of different monikers, but the key to understanding them is knowing that they are really just cash bonus plans, where the cash amounts are determined by reference to a company’s stock. Phantom equity can have significant value, but may be perceived as less valuable by workers because of the contractual nature of the promises. Phantom equity plans can be set up as purely discretionary bonus plans, which is less attractive than owning a piece of something. Two examples of phantom equity are phantom stock and stock appreciation rights:
+	- A [**phantom stock**](http://www.investopedia.com/articles/stocks/12/introduction-phantom-stock.asp) award entitles you to a payment equal to the value of a share of the company’s stock, upon the occurrence of certain events.
 	- [**Stock appreciation rights (SAR)**](https://www.nceo.org/articles/phantom-stock-appreciation-rights-sars) give the recipient the right to receive a payment calculated by reference to the appreciation in the equity of the company.
 - [**Warrants**](https://en.wikipedia.org/wiki/Warrant_%28finance%29) are another kind of option to purchase stock, generally used in investment transactions (for example, in a convertible note offering, investors may also get a warrant; or a law firm may ask for one in exchange for vendor financing). As an employee or advisor, you may not encounter warrants, but it’s worth knowing they exist. They differ from stock options in that they are more abbreviated and stand-alone legal documents, not granted pursuant to a “plan.” Also, because they are usually used in the investment context, they do not typically include service-based vesting provisions or termination at end of service, and are valid for a set number of years (often, 10 years).
 
-## Equity Compensation Details
+### How equity compensation is granted
 
 Now for the details around how stock and options are used for compensation.
 
 - Companies can give equity compensation as stock awards, stock options, or RSUs. While the intent of each is similar, they [differ in many ways](http://joewallin.com/2014/09/13/rsus-vs-restricted-stock-vs-stock-options/), particularly around taxation. RSUs generally don’t make sense for early stage companies.
 - If companies do grant stock, it may be **restricted stock**. In this context, “restricted” refers to the fact that the stock will be subject to repurchase at a lower rate than fair market value or cost, and the repurchase right lapses over the service-based vesting period.
```

Comment:
This sentence is a very hard for me to parse. I actually don't know what it means. Does it mean that, if I buy my stock for $100, it's possible that my employer could buy it back for $50? That doesn't seem fair. How does that work? Is that answered later? "the repurchase right lapses over the service-based vesting period" reads at a post-graduate reading level here.

Reply:
🚧